common/djangoapps/util/tests/test_db.py

```diff
@@ -198,6 +198,9 @@ class MigrationTests(TestCase):
     """
 
     @override_settings(MIGRATION_MODULES={})
+    @unittest.skip(
+        "Temporary skip for https://openedx.atlassian.net/browse/PROD-1886 where a column is to be removed"
+    )
```

Comment:
Just for my knowledge, can you explain the context for this?

Reply:
https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations#EverythingAboutDatabaseMigrations-Deploymentandbackward-compatiblemigrations mentioned in the doc that if change under platform, we can skip the test.